source/manual/migrate-testing-from-phantomjs-to-selenium-chrome.html.md

```diff
@@ -0,0 +1,17 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Migrate testing from PhantomJS to Selenium/headless Chrome
+section: Testing
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-07-10
+review_in: 6 months
```

Comment:
I wonder if we should review this sooner as we probably want to remove these docs once everything has migrated.

Reply:
I've given it 2 months from today which should be enough time